ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -65,6 +65,52 @@ tests:
         VIP_DHCP_ALLOCATION=no
         PROXY=yes
     workflow: baremetalds-assisted-install
+- as: e2e-metal-assisted-none-platform
+  steps:
+    cluster_profile: packet
+    dependencies:
+      ASSISTED_SERVICE_IMAGE: pipeline:assisted-service
+    env:
+      ASSISTED_CONFIG: |
+        PLATFORM=none
```

Comment:
Aren't you trying to install with DHCP?  I don't think it is supported for none platform

Reply:
DHCP is enabled by default